frontend/src/components/admin/schedule/AdminScheduleVolunteerTable.tsx

```diff
@@ -1,37 +1,30 @@
 import React, { useState, useEffect } from "react";
-import {
-  Flex,
-  VStack,
-  Text,
-  Button,
-  Spacer,
-  useBoolean,
-} from "@chakra-ui/react";
+import { Flex, VStack, Text, Button, Spacer } from "@chakra-ui/react";
 
 import AdminScheduleVolunteerRow from "./AdminScheduleVolunteerRow";
 import { SignupsAndVolunteerGraphQLResponseDTO } from "../../../types/api/SignupTypes";
 
 type AdminScheduleVolunteerTableProps = {
   signups: SignupsAndVolunteerGraphQLResponseDTO[];
   currentlyEditingSignups: SignupsAndVolunteerGraphQLResponseDTO[];
-  onEditSignupsClick: (
-    signups: SignupsAndVolunteerGraphQLResponseDTO[],
-  ) => void;
   onSelectAllSignupsClick: () => void;
   onSignupCheckboxClick: (id: string, isChecked: boolean) => void;
+  isEditing: boolean;
+  onButtonClick: () => void;
```

Comment:
nit: Maybe change name into something more meaningful
```suggestion
  onEditSaveClick: () => void;
```